third_party/Makefile

```diff
@@ -41,3 +41,8 @@ ifneq (,$(wildcard nccl-tests/Makefile))
 	cd ./nccl-tests && make MPI=1 MPI_HOME=$(MPI_HOME_PATH) -j
 	cp -v ./nccl-tests/build/* $(SB_MICRO_PATH)/bin/
 endif
+
+# Build FIO from commit 0313e9 (fio-3.27 tag).
+fio: sb_micro_path
+ifneq (,$(wildcard fio/Makefile))
+	cd ./fio && ./configure --prefix=$(SB_MICRO_PATH) && make -j && make install
```

Comment:
Misssing endif, the build pipeline failed:
```
#16 0.246 make: Entering directory '/opt/superbench/third_party'
#16 0.246 make: Leaving directory '/opt/superbench/third_party'
#16 0.248 Makefile:49: *** missing 'endif'.  Stop.
```

Reply:
fixed